### ISSUE TITLE
Buttons Missing in Safari

### ISSUE THREAD

**Issue by bkonia** [pojome/elementor#4512]
I'm having a strange issue with Elementor in Safari 11.1. When I hover over a section, it displays the Edit Section control at the top of the section, but it's missing all the other controls (Duplicate, Add, Save, Remove). When I view the same page in Chrome, all the buttons are visible. I've attached screen shots showing the same section displayed in both Safari and Chrome.

![2018-05-22_10-18-43](https://user-images.githubusercontent.com/2266423/40379089-1b397396-5daa-11e8-8cf5-ebf98d475ff2.png)
![2018-05-22_10-17-20](https://user-images.githubusercontent.com/2266423/40379095-1ca14948-5daa-11e8-828b-2842bba0e259.png)

**Comment by bainternet** [comment]
Thanks for reporting,
Please follow guidelines for opening a new issue/feature request, so we can help you better.
This report is missing key information: 

* Prerequisites.
* Steps to replicate.
* Isolating the problem.
* System info.

**Comment by bkonia** [comment]
- Prerequisites: A website with WordPress 4.9.6 and Elementor Pro 2.0.7 installed.

- Steps to replicate:
    1. Create a page in Elementor
    2. Add a new section
    3. There is no step 3. As I demonstrated in the previously attached screen shots, after adding the section, only the Edit Section control is visible. Duplicate, Add, Save, Remove buttons are not present.

- Isolating the Problem: The problem is specific to Safari. It works fine in other browsers.

- System Info: As I already mentioned, Safari 11.1. Obviously, it's a Mac.

**Comment by klopfer** [comment]
I see this issue is closed.  But I am having the same problem.  Works fine in chrome.  Not in safari. Developer console does not report any errors.  Cleared cache.  Forced reload.  Same result.  

Create a site.  Create a page.  Edit it Elementor in safari and you see this result.

**Comment by bkonia** [comment]
Right. I gave up on trying to use Elementor in Safari. There's obviously a problem, but the developers choose to ignore it.
